FoodBookApp/Handlers/NotificationHandler.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  NotificationHandler.swift
+//  FoodBookApp
+//
+//  Created by Laura Restrepo on 17/03/24.
+//
+
+import Foundation
+import UserNotifications
+
+class NotificationHandler {
+    func askPermission() {
+        UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .badge, .sound])
+        { success, error in
+            if success {
+                print("Access granted!")
+            } else if let error = error {
+                print(error.localizedDescription)
+            }
+        }
+    }
+    
+    func cancelNotification(identifier: String) {
+        UNUserNotificationCenter.current().removePendingNotificationRequests(withIdentifiers: [identifier])
+    }
+    
+    func sendLastReviewNotification(date: Date) {
+        
+        let notificationIdentifier = "lastReviewNotification"
+
+        cancelNotification(identifier: notificationIdentifier)
+                    
+        let triggerDate = Calendar.current.date(byAdding: .day, value: 4, to: date)!
```

Comment:
Change this to `let triggerDate = Calendar.current.date(byAdding: .second, value: 10, to: date)!` for tests